api/datastore/datastore.go

```diff
@@ -40,6 +40,11 @@ func NewStore(storePath string, fileService portainer.FileService, connection po
 func (store *Store) Open() (newStore bool, err error) {
 	newStore = true
 
+	encryptionReq := store.connection.IsEncryptionRequired()
```

Comment:
Still not sure about this function name.

Reply:
how about `DoesStoreNeedEncryption`?